ports/geckolib/glue.rs

```diff
@@ -4552,7 +4552,16 @@ pub unsafe extern "C" fn Servo_SelectorList_Drop(list: RawServoSelectorListOwned
 fn parse_color(value: &str) -> Result<specified::Color, ()> {
```

Comment:
Is every use of this function intended to support full CSS syntax, including `calc`? I traced one use back to `aBGColor` in `DocumentRendererChild::RenderDocument`. Is that `<body bgcolor>`? https://html.spec.whatwg.org/multipage/#the-page links to https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#rules-for-parsing-a-legacy-colour-value

Reply:
It is not, that's handled from `nsAttrValue::ParseColor` (https://searchfox.org/mozilla-central/rev/0b613c3887789f7786cd3131dfe9648398f4a6ac/dom/html/HTMLBodyElement.cpp#58).

That seems to be the default background color to render in a child process if there's no content.

The only web-exposed use of this is `CanvasGradient::AddColorStop`, and the spec for that is:

  https://html.spec.whatwg.org/multipage/#dom-canvasgradient-addcolorstop

Which says to parse as a CSS `<color>` value, that is, with `calc()`.